### ISSUE TITLE
Hosting for SchemaSpy (currently hosted on phoenix.dataverse.org)

### ISSUE THREAD

**Issue by pdurbin** [IQSS/dataverse#7031]
Long ago I configured http://phoenix.dataverse.org/schemaspy/latest/index.html to host schema information about the Dataverse database. I often link to this site when helping people understand something about the database. I also wrote it up in the dev guide at http://guides.dataverse.org/en/4.20/developers/tips.html#schemaspy

<img width="973" alt="Screen Shot 2020-06-26 at 4 11 37 PM" src="https://user-images.githubusercontent.com/21006/85897352-2593b200-b7c8-11ea-902c-4571a8b42a5d.png">

The phoenix server is being shut down at some point so we need to find alternative hosting for these docs.

As indicated in the screenshot above, the script to build the SchemaSpy docs can be found at https://github.com/IQSS/dataverse/blob/v4.20/scripts/deploy/phoenix.dataverse.org/post#L15 . It's a oneliner: `java -jar downloads/schemaSpy_5.0.0.jar -t pgsql -host localhost -db dvndb -u postgres -p secret -s public -dp scripts/installer/pgdriver/postgresql-42.2.9.jar -o /var/www/html/schemaspy/latest`

**Comment by donsizemore** [comment]
@pdurbin Permanent web-hosting would be great, but this strikes me as an excellent workflow to Dockerize.

**Comment by pdurbin** [comment]
People seem to like the SchemaSpy diagrams. @BPeuch just [said][], "Wow, that ER diagram of Dataverse's database rocks, Philip! Thanks for posting it."

One thing I didn't mention when I opened this issue is that builds are no longer happening on the phoenix server so the diagrams hosted there are increasing out of date as the database evolves.

[said]: https://groups.google.com/g/dataverse-community/c/Bm6nQ5fk6sQ/m/8wwIFvPsBAAJ

**Comment by BPeuch** [comment]
That's good to know, @pdurbin. Thanks for tagging me. We hope an alternative representation system can be found.

**Comment by donsizemore** [comment]
@pdurbin if it just needs to live somewhere, how about off the root of guides.dataverse.org?

**Comment by donsizemore** [comment]
@BPeuch I generated this today from a Dataverse 5.4 DB, if you'd like to take a look: https://guides.dataverse.org/en/5.4/schemaspy/

**Comment by djbrooke** [comment]
Thanks @donsizemore, this is great. I talked with @pdurbin about this a little while ago and he confirmed to me it was most important to have it for a specific version - I was thinking that it needed to be provided on demand for any branches in progress but he said this isn't usually the case. I think if it could be included on the guides server I think it'd be great, thanks for adding it for 5.4!

**Comment by pdurbin** [comment]
Yeah, @donsizemore and I were discussing this yesterday and I've very happy that once again there is a publicly accessible version of the schemaspy output (I had built my own local version for now). Thanks, Don!

**Comment by BPeuch** [comment]
Oh this is very practical! Thanks a lot @donsizemore!

I like the network (ER) model, but this is also very handy, especially the neat presentation of columns with the sorting out of IDs and other such categories. :wrench:

**Comment by pdurbin** [comment]
@donsizemore published a version for 5.14: https://guides.dataverse.org/en/5.14/schemaspy/

Thank you, Don!

See also https://groups.google.com/g/dataverse-community/c/f95DQU-wlVM/m/9HcRbaSVBQAJ

**Comment by poikilotherm** [comment]
This reminds me of https://blog.esciencecenter.nl/versioned-documentation-using-only-github-actions-and-github-pages-1825296e31aa and how we could make such a thing to not just host our guides on Pages, but add OpenAPI docs, SchemaSpy, JSON Schemas etc etc etc.

**Comment by cmbz** [comment]
To focus on the most important features and bugs,     we are closing issues created before 2020 (version 5.0)     that are not new feature requests with the label 'Type: Feature'.    

If you created this issue and you feel the team     should revisit this decision, please reopen the issue and leave a comment.

**Comment by pdurbin** [comment]
Yes, I still think we should give our valuable SchemaSpy information a good home, so I'll reopen this. I guess I'll give it a size of 10 but that's a guess.

**Comment by pdurbin** [comment]
> @donsizemore published a version for 5.14: https://guides.dataverse.org/en/5.14/schemaspy/

I'd be remiss if I didn't point out that @donsizemore continues to publish SchemaSpy data to the guides server (thank you!) from time to time.

He doesn't publish for every release (the latest was https://guides.dataverse.org/en/6.2/schemaspy/index.html I believe) but I always try to [let the community know](https://groups.google.com/g/dataverse-community/c/f95DQU-wlVM/m/pbJcSaTKAAAJ).

Should we continue with publishing SchemaSpy to the guides server? If so, perhaps we could add a manual step for now to a future versions of our [making releases](https://guides.dataverse.org/en/6.3/developers/making-releases.html) doc. Then automate it?

@donsizemore what do you think?

**Comment by donsizemore** [comment]
@donsizemore it's easy to do and I'm happy to do it; it's part of the Dataverse-Ansible run but not automated to push to guides.dataverse.org. I just zip it up and copy it over when I remember to.

**Comment by pdurbin** [comment]
@donsizemore these days, we remind you as part of our release process! Thanks!

I just created this PR to close this issue:

- #11291

Please feel free to take a look!
